Mathlib/RingTheory/Ideal/Maps.lean

```diff
@@ -334,35 +334,42 @@ end Injective
 
 /-- If `f : R ≃+* S` is a ring isomorphism and `I : Ideal R`, then `map f.symm (map f I) = I`. -/
 @[simp]
-theorem map_of_equiv (I : Ideal R) (f : R ≃+* S) :
+theorem map_of_equiv {I : Ideal R} (f : R ≃+* S) :
```

Comment:
Why making `I` implicit? It is not mentioned in other assumptions.

Reply:
I find that `I` can always be inferred in current practice, and it can be written as `I.map_of_equiv` if it is really necessary to specify `I`.

Reply:
OK, no worries.